src/styles/card.scss

```
@@ -0,0 +1,119 @@
+body {
+  margin: 0;
+}
+
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+
+  @include number-of-colums;
+}
+
+.card {
+  display: block;
+  font-family: Roboto, sans-serif;
+  color: $main-text-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: $transition-duration-value;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding-inline: 19px;
+    padding-top: 32px;
+  }
+
+  &__main {
+    padding-inline: $content-padding-inline;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin-top: 36px;
+    margin-bottom: 0;
+    transition-duration: $transition-duration-value;
+  }
+
+  &:hover &__product-name {
+    color: #34568b;
+  }
+
+  &__product-code {
+    color: $second-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    color: $second-text-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin-top: 24px;
    
    &-text {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```